src/auth/auth.service.ts

```diff
@@ -16,16 +16,13 @@ export class AuthService {
     const user = await this.userService.findOne(name);
 
     if (!user) {
-      throw new HttpException(
-        'There is no user with such a name',
-        HttpStatus.NOT_FOUND,
-      );
+      throw new HttpException('Wrong credentials', HttpStatus.NOT_FOUND);
```

Comment:
Nest has a build in Http errors that you should use. 
https://docs.nestjs.com/exception-filters
Also, the status is incorrect.
In case of incorrect credentials it should be Unauthorized